src/main/groovy/org/transmartproject/batch/highdim/jobparams/StandardHighDimDataParametersModule.groovy

```diff
@@ -52,6 +50,14 @@ class StandardHighDimDataParametersModule
 
     void munge(ExternalJobParametersInternalInterface ejp)
             throws InvalidParametersFileException {
+        if (ejp[LOG_BASE] == null) {
+            ejp[LOG_BASE] = '2'
+        }
+
+        if (ejp[DATA_TYPE] == null) {
+            ejp[DATA_TYPE] = 'R'
+        }
```

Comment:
This makes `DATA_TYPE` user settable, but the only supported value is `R`, which is not validated.